src/source-dpdk.c

```diff
@@ -398,7 +443,27 @@ static TmEcode ReceiveDPDKLoop(ThreadVars *tv, void *data, void *slot)
                 TmThreadsCaptureHandleTimeout(tv, NULL);
                 last_timeout_msec = msecs;
             }
-            continue;
+
+            if (!ptv->intr_en)
```

Comment:
what is the impact of this new block in what looks like the critical loop of our dpdk support?

Reply:
During my recent testing, I found no visible impact on the performance.